### ISSUE TITLE
Consistent Upload Connection Timeout

### ISSUE THREAD

**Issue by tsujp** [multiplegeorges/vue-cli-plugin-s3-deploy#25]
When I try and upload a simple Vue3 project to my S3 bucket all files upload nicely except for a single `.js` file that refuses to play nicely and consistently causes the following timeout errors:

```
 ERROR  Upload failed: js/chunk-vendors.eb7dc48c.js
 ERROR  TimeoutError: Connection timed out after 10000ms
 ERROR  TimeoutError: Connection timed out after 10000ms
```

All settings are default on `Mac OS High Sierra 10.13.5` with `vue-cli-plugin-s3-deploy 2.1.1` using `aws 1.16.10` using `Python 3.7.0` both installed via Homebrew.

These are the `vue.config.js` settings

```
module.exports = {
  pluginOptions: {
    s3Deploy: {
      region: 'ap-southeast-1',
      bucket: '[NAME-REDACTED]',
      assetPath: 'dist',
      deployPath: '/',
      pwa: false,
      enableCloudfront: false,
      uploadConcurrency: 3
    }
  }
}
```

All region settings are correct, this is what an upload consistently looks like:

```
 INFO  (1/6) Uploaded index.html
 INFO  (2/6) Uploaded js/app.d39be2a8.js
 INFO  (3/6) Uploaded js/app.d39be2a8.js.map
 INFO  (4/6) Uploaded css/app.94f00a75.css
 ERROR  Upload failed: js/chunk-vendors.eb7dc48c.js
 ERROR  TimeoutError: Connection timed out after 10000ms
 ERROR  TimeoutError: Connection timed out after 10000ms
 INFO  (5/6) Uploaded js/chunk-vendors.eb7dc48c.js.map
```

Here's an `ls -lh` of the `dist/`, `dist/css/`, and `dist/js/` directories respectively so you can see the file sizes (all small)

```
drwxr-xr-x  3 tsujp  staff    96B 16 Sep 19:42 css
-rw-r--r--  1 tsujp  staff   695B 17 Sep 01:12 index.html
drwxr-xr-x  6 tsujp  staff   192B 16 Sep 19:42 js

-rw-r--r--  1 tsujp  staff   253B 17 Sep 01:12 app.94f00a75.css

-rw-r--r--  1 tsujp  staff   3.9K 17 Sep 01:12 app.d39be2a8.js
-rw-r--r--  1 tsujp  staff    19K 17 Sep 01:12 app.d39be2a8.js.map
-rw-r--r--  1 tsujp  staff   905K 17 Sep 01:12 chunk-vendors.eb7dc48c.js
-rw-r--r--  1 tsujp  staff   3.6M 17 Sep 01:12 chunk-vendors.eb7dc48c.js.map
```

**Comment by nicekiwi** [comment]
How large are the files you're uploading? 

Might be worth adding an option to this plugin for a longer timeout, currently only 10 seconds, which is a bit small IMHO.

**Comment by multiplegeorges** [comment]
@nicekiwi Yes, that's the likely culprit.

**Comment by tsujp** [comment]
Shown in the `ls -lh` at the bottom the files are very small, and I played around with adding the timeout to crazy amounts like 30 seconds with issues still; no timeout amount solved the problem. `chunk-vendors.eb7dc48c.js.map` is 3.6 mb and uploads fine, yet `chunk-vendors.eb7dc48c.js` which is 905 kb is the one failing.

**Comment by multiplegeorges** [comment]
@tsujp I just published 3.0.0. Could you give it a shot with this new version?

**Comment by nicekiwi** [comment]
Closing due to lack of activity.
